base/applications/mspaint/imgarea.cpp

```diff
@@ -76,26 +76,26 @@ LRESULT CImgAreaWindow::OnSize(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
                0,
                0, 3, 3, TRUE);
     sizeboxCenterTop.MoveWindow(
-               imgXRes * toolsModel.GetZoom() / 2000 + 3 * 3 / 4,
+               toolsModel.Zoomed(imgXRes / 2) + 3 * 3 / 4,
```

Comment:
Would it be more correct to do `toolsModel.Zoomed(imgXRes) / 2` (plus the rest) instead?

Reply:
Fixed.